package.json

```diff
@@ -250,7 +250,7 @@
     "lint": "aegir lint",
     "build": "aegir build",
     "release": "aegir release",
-    "test": "npm run lint && npm run test:node && npm run test:chrome && npm run test:ts",
+    "test": "npm run test:node && npm run test:chrome && npm run test:ts",
```

Comment:
```suggestion
    "test": "npm run lint && npm run test:node && npm run test:chrome && npm run test:ts",
```

I agree that `no-only-tests` is kind of gross, I didn't know that we're not stuck with that thanks to the move to aegir! But `npm run test:node` is still available for that case and my personal preference is still to catch linting errors before folks push a PR—getting the PR right the first time is the best way to get something merged, the chances of getting a PR modified by the submitter lowers with each interaction.